presto-kinesis/src/main/resources/META-INF/services/com.facebook.presto.spi.Plugin

```diff
@@ -0,0 +1 @@
+KinesisPlugin
```

Comment:
Remove this file. It will be created automatically by the `presto-plugin` Maven packaging

Reply:
removed